src/bullsAndCowsValidator.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+function getHint(secret, guess) {
```

Comment:
Consider renaming the function 'getHint' to something more descriptive like 'calculateBullsAndCows'.